tests/cli/upload/test_cli_upload_observations.py

```diff
@@ -107,3 +108,26 @@ def test_get_observations_api(cg_context: CGConfig, helpers: StoreHelpers):
     # THEN a MIP-DNA API should be returned
     assert observations_api
     assert isinstance(observations_api, MipDNAObservationsAPI)
+
+
+def test_get_observations_api_raredisease(cg_context: CGConfig, helpers: StoreHelpers):
```

Comment:
I wouldn't create another test for rare disease and would instead replace the one we have for MIP with the new workflow. We are just testing here that the method performs as expected in general

Reply:
I would keep both tests because MIP will be run in parallel with raredisease for a little while.

Reply:
But my point is that I don't think it brings any value to have multiple tests covering the same part of the code. So, we should either not add this test or replace the one we have for mip with the one for raredisease